### PR TITLE
Add support for Placeholders in ListItem w/ TextInput

### PR DESCRIPTION
**Pull request by lax1n**
PR for issue #509

### CODE REVIEW

**Comment by iRoachie** [comment]
@lax1n Awesome PR, could you update the docs as well. So others would know the prop exist.

File: example/src/views/lists_home.js [hunk 1]
```diff
@@ -192,6 +192,21 @@ class Icons extends Component {
...
+          />
+          <ListItem
+            onPress={log}
+            title={'Name w/ placeholder'}
+            textInput={true}
+            textInputPlaceholder={'John Doe'}
+            hideChevron
+          />
+        </List>
+        <List>
```
Comment (Monte9) [inline example/src/views/lists_home.js]: 💯 for adding implementation code in the `/example` app.

**Comment by Monte9** [comment]
@lax1n `v1` should have a much cleaner ListItem component. But in the meantime, this should serve as a good fix. Thanks. 👍

**Comment by avaldivi** [comment]
I'm not sure if this is the right place and I apologize if it's not but is there a prop available for placeholder color? I have a dark background and unfortunately placeholder is by default a dark color. Am I missing something in the docs?

```
        <ListItem
          hideChevron
          textInput
          textInputStyle={{ color: constants.white }}
          textInputPlaceholder={portfolioStore.initialBalance}
          textInputOnChangeText={this.onChangeText}
          textInputValue={this.state.newBalanceText}
          title="Starting Balance"
          // containerStyle={{ backgroundColor: constants.white }}
        />
```

screen shot 2017-10-10 at 3 26 48 pm

**Comment by Monte9** [comment]
@avaldivi sorry about the delayed response. 

Yes, you can absolutely do it. You could use the `titleStyle` to add additional styling to the text on the ListItem.
```
titleStyle | none | object (style) | additional title styling (optional)
```

Hope this helps.

**Pull request merged**